packages/0x.js/src/types.ts

```diff
@@ -397,9 +397,11 @@ export interface JSONRPCPayload {
 }
 
 /*
+ * orderExpirationCheckingIntervalMs: How often to check for expired orders
  * eventPollingIntervalMs: How often to poll the Ethereum node for new events
```

Comment:
I think for both of these we should add what the default values are set to. Thoughts?